pandas/tests/generic/test_frame.py

```diff
@@ -269,3 +269,28 @@ def test_deepcopy_empty(self):
         empty_frame_copy = deepcopy(empty_frame)
 
         self._compare(empty_frame_copy, empty_frame)
+
+    @pytest.mark.parametrize("header, expected", [
+        (False, 'index.name,,\n0,0,0\n1,0,0\n'),
+        (True, 'index.name,0,1\n0,0,0\n1,0,0\n')
+    ])
+    def test_to_csv_header_not_multi_index(self, header, expected):
```

Comment:
Instead of "not_multi_index," just call it "single_index"

Reply:
You could potentially even squash your tests together so that in your parameterization, you pass in the `index` that you want to set on `df.index`...not sure if that makes parameterization a little more complicated, but that's also another thought, given how similar how some of the code is between both of your tests.

Reply:
yeah, thanks for the review! @gfyoung i thought about it, but i thought they represented two different cases although the ultimate idea is to test if index name is displayed or not... so i separated them... but i will try to squash them a bit and see if it makes tests clearer.